### PR TITLE
fixing issues with wallet-app integration

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: packages/react-native/lib/index.js [hunk 1]
```diff
@@ -92,42 +96,46 @@ export function useAccount(address) {
...
-  const sandboxWebViewRef = useRef();
-  const baseUrl =
-    Platform.OS === 'ios' ? 'app-html' : 'file:///android_asset/app-html';
+  const [firstFetch, setFirstFetch] = useState();
+  const networkId = useMemo(() => {
+    return wallet?.getNetworkId();
+  }, [wallet]);
+
+  console.log('Current networkId', networkId);
+  console.log('wallet documents', documents);
```
Comment (mike-parkhill) [inline packages/react-native/lib/index.js:114]: we probably don't want to dump all the documents out - that could be pretty large, right? and possibly hold PII

File: packages/react-native/lib/wallet.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  if (!wallet) {
+    wallet = await initializeWallet();
+  }
+
+  return wallet;
+}
+export async function initializeWallet() {
+  wallet = await createWallet({
+    databasePath: 'dock-wallet',
+    dbType: 'react-native',
```
Comment (mike-parkhill) [inline packages/react-native/lib/wallet.ts:24]: I wouldn't expect a `dbType` to be `react-native`. I was expecting a database related value like postgres or mysql or sqlite or something

Comment (maycon-mello) [reply packages/react-native/lib/wallet.ts:24]: I agree, but this is the constant value defined on `typeorm` side. https://github.com/typeorm/react-native-example/blob/master/src/App.tsx#L42

`react-native` is used for mobile a sqlite, and is mentioned on their docs. For nodejs `sqlite` should be used. We could map it out on our side, but could create breaking changes and not be consistent with `typeorm` docs

Comment (mike-parkhill) [reply packages/react-native/lib/wallet.ts:24]: ah, that's an odd choice on their part

no need to complicate things on our side

File: packages/react-native/lib/index.js (hunk 1 above)
Comment (maycon-mello) [reply packages/react-native/lib/index.js]: Fixed

File: packages/wasm/lib/services/credential/service.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ import {verifyCredential} from '@docknetwork/sdk/utils/vc/credentials';
 import {PEX} from '@sphereon/pex';
 import {keyDocToKeypair} from './utils';
 import {getDock} from '../dock/service';
+import {Logger} from '../../core/logger';
```
Comment (mike-parkhill) [inline packages/wasm/lib/services/credential/service.js:18]: lint

**Review by mike-parkhill — APPROVED** [review]

**Comment by mike-parkhill** [comment]
Still has a lint/test failure but otherwise is good

File: integration-tests/switch-wallet.test.ts [hunk 4]
```diff
@@ -36,14 +36,14 @@ describe('Switch wallet', () => {
...
 
-  it('expect to filter mainnet documents from testnet documents', async () => {
-    await setNetwork('mainnet');
-    await createWalletData();
-    await assertWalletData();
-    await setNetwork('testnet');
-    const isEmpty = await isWalletEmpty();
-    expect(isEmpty).toBe(true);
-  });
+  // it('expect to filter mainnet documents from testnet documents', async () => {
```
Comment (mike-parkhill) [inline integration-tests/switch-wallet.test.ts:39]: is there a plan for restoring this test?

Comment (maycon-mello) [reply integration-tests/switch-wallet.test.ts:39]: Yes, https://dock-team.atlassian.net/browse/DCKA-1708. I will fix those today

Comment (maycon-mello) [reply integration-tests/switch-wallet.test.ts:39]: I will push another PR to fix that

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**